File: CodeForces/Codeforces Round 871 (Div. 4)/a.cpp
v#include <iostream>
#include <string>
using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--) {
        string s;
        cin >> s;
        string target = "codeforces";
        int diff = 0;
        for (int i = 0; i < 10; i++) {
            if (s[i] != target[i]) {
                diff++;
            }
        }
        cout << diff << endl;
    }
    return 0;
}

File: CodeForces/Codeforces Round 871 (Div. 4)/a.py
t = int(input())

for i in range(t):
    s = input()

    # Initialize the target string
    target = "codeforces"

    # Initialize the number of differences to 0
    diff = 0

    # Loop through each character in the strings
    for j in range(10):
        # Check if the characters differ
        if s[j] != target[j]:
            # Increment the number of differences
            diff += 1

    # Output the number of differences
    print(diff)
